internal/handlers/pg/msg_update.go

```diff
@@ -223,7 +234,7 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 func (h *Handler) update(ctx context.Context, sp pgdb.SQLParam, doc *types.Document) (int64, error) {
```

Comment:
I'd say here, and in the callers of that func we need to refactor it to `sp *pgdb.SQLParam` too.